LICENSE

```diff
@@ -219,3 +219,9 @@ The text of each license is the standard Apache 2.0 license.
 
 mvnw files from https://github.com/apache/maven-wrapper Apache 2.0
 streampark-console/streampark-console-service/src/main/assembly/bin/setclasspath.sh from https://github.com/apache/tomcat
+
+========================================================================
+MIT License
+========================================================================
+
+The files under the directory streampark-console/streampark-console-webapp/ without the "Apache License 2.0" License header are provided under the MIT License, from https://github.com/vbenjs/vue-vben-admin
```

Comment:
You should explicitly list the files.

Reply:
> You should explicitly list the files or directories.

Okay,  Thanks for your review , we will list the files and directories

Reply:
We have listed the copied files and directories, If you have time, please double-check. Thanks very much! 😊